docs/reference/api-docs.asciidoc

```diff
@@ -504,6 +504,8 @@ HTTPConfig holds the HTTP layer configuration for resources.
 
 
 
+
+
```

Comment:
Do you know why these new lines are added?

Reply:
@thbkrkr 936cf510c specifically was to address this. Something is "requiring" those new spaces, and one of the checks (I think the "local changes" check) tagged this, so I specifically added those spaces to satisfy that. Lmk if you want me to dig further, or just accept it.